app/main.ipynb

```diff
@@ -130,7 +139,8 @@
    "outputs": [],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "selected_elements = array[[1, 2, 3, 4, 5]]\n",
+    "print(int(selected_elements.sum()))# write your code here"
```

Comment:
The selected elements from the array do not sum up to 27. Please check the indices used in the multi-indexing to ensure the correct elements are selected.